rq/worker.py

```diff
@@ -1605,9 +1605,15 @@ def handle_job_success(self, job: 'Job', queue: 'Queue', started_job_registry: S
                     assert job.started_at
                     assert job.ended_at
                     time_taken = job.ended_at - job.started_at
-                    self.log.info(
-                        'Successfully completed %s job in %ss on worker %s', job.description, time_taken, self.name
```

Comment:
I think we should simply change this to `'Successfully completed job %s in %ss on worker %s', job.id, time_taken, self.name`. Showing job ID would make it easier for someone to trace a particular job's execution path.

Reply:
Ok. We can apply that. Would you like to print yet the description if allowed by user?

Reply:
For backwards compatibility's sake, yes.

Reply:
Done!